src/components/EmojiPicker/EmojiPickerMenu/BaseEmojiPickerMenu.js

```diff
@@ -112,6 +112,10 @@ function BaseEmojiPickerMenu({headerEmojis, scrollToHeader, isFiltered, listWrap
     const styles = useThemeStyles();
     const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
 
+    // related to https://github.com/Expensify/App/issues/34522
+    const roundedWindowWidth = Math.floor(windowWidth);
+    const containerWidth = isSmallScreenWidth ? roundedWindowWidth : CONST.EMOJI_PICKER_SIZE.WIDTH;
```

Comment:
```suggestion
    const containerWidth = isSmallScreenWidth ? Math.floor(windowWidth) : CONST.EMOJI_PICKER_SIZE.WIDTH;
```

Also add comment explain why we need to truncate the decimal values